ethereum/core/src/main/java/tech/pegasys/teku/core/signatures/LocalMessageSignerService.java

```diff
@@ -30,32 +31,38 @@ public LocalMessageSignerService(final BLSKeyPair keypair, final AsyncRunner asy
   }
 
   @Override
-  public SafeFuture<BLSSignature> signBlock(final Bytes signingRoot) {
+  public SafeFuture<BLSSignature> signBlock(
+      final Bytes signingRoot, final Map<String, Object> additionalProperties) {
     return sign(signingRoot);
   }
 
   @Override
-  public SafeFuture<BLSSignature> signAttestation(final Bytes signingRoot) {
+  public SafeFuture<BLSSignature> signAttestation(
+      final Bytes signingRoot, final Map<String, Object> additionalProperties) {
     return sign(signingRoot);
   }
 
   @Override
-  public SafeFuture<BLSSignature> signAggregationSlot(final Bytes signingRoot) {
+  public SafeFuture<BLSSignature> signAggregationSlot(
+      final Bytes signingRoot, final Map<String, Object> additionalProperties) {
     return sign(signingRoot);
   }
 
   @Override
-  public SafeFuture<BLSSignature> signAggregateAndProof(final Bytes signingRoot) {
+  public SafeFuture<BLSSignature> signAggregateAndProof(
```

Comment:
This and the below sign methods don't require any additional parameters for the signing protection it's only the attestation and block

Reply:
fixed in ExternalSigner.